HiP-DataStore.Model/Rest/ReviewUpdateArgs.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+using static PaderbornUniversity.SILab.Hip.DataStore.Model.Entity.Review;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Rest
+{
+    public class ReviewUpdateArgs
+    {
+        public bool Approved { get; set; }
+
+        public string Description { get; set; }
+
+        public string Comment { get; set; }
+
+        /// <summary>
+        /// Amount of students, that need to approve, in order to approve the review
+        /// </summary>
+        public int? StudentsToApprove { get; set; }
+
+        public bool? ReviewableByStudents { get; set; }
+
+        public List<string> Reviewers { get; set; }
+
+        [JsonIgnore]
+        public string EntityType { get; set; }
+
+        [JsonIgnore]
+        public int EntityId { get; set; }
+
+        [JsonIgnore]
+        public List<Comment> Comments { get; set; } = new List<Comment>();
```

Comment:
It might be more feasible to create a separate resource type for ReviewComments. With the current situation you would have to pass the whole Review object every time a comment was added. So I suggest we change this to a list of ints which are then the ids for the comments